docs/Release_Guide/metviewer_development.rst

```diff
@@ -7,5 +7,13 @@ Create a new vX.Y.Z-betaN or vX.Y.Z-rcN development release from the develop bra
 
 .. include:: release_steps/open_release_issue.rst
 .. include:: release_steps/clone_project_repository.rst
-.. include:: release_steps/checkout_main_branch.rst
-.. include:: release_steps/create_release_feature_branch.rst
+.. include:: release_steps/checkout_develop_branch.rst
+.. include:: release_steps/metviewer/update_version.rst
+.. include:: release_steps/update_release_notes_development.rst
+.. include:: release_steps/metviewer/update_docs_develop.rst
```

Comment:
This line ".. include:: release_steps/metviewer/update_docs_develop.rst"
should be removed because we have moved to ReadtheDocs and the steps listed in update_docs_develop.rst are no longer relevant.